flocker_bb/builders/maint.py

```diff
@@ -196,6 +216,9 @@ def getBuilders(slavenames):
             # These slaves are dedicated slaves.
             slavenames=slavenames['fedora-20/vagrant'],
             factory=makeCleanOldAcceptanceInstances()),
+        # FLOC-2288
+        # Make a builder config using CleanVolumes
+        # It can run on the same slave as the above
```

Comment:
I wonder if this should be a separate builder, or if it should be the same. I think I could be convinced either way.